tests/integration/conftest.py

```diff
@@ -29,19 +41,25 @@ def cognite_client():
             credentials=credentials,
         )
     )
+
 @pytest.fixture(scope="session")
 def azure_credential():
     return DefaultAzureCredential()
 
 @pytest.fixture(scope="session")
 def lakehouse_timeseries_path(azure_credential):
-    lakehouse_timeseries_path = os.environ["LAKEHOUSE_ABFSS_PREFIX"] + "/Tables/" + TEST_TIMESERIES_TABLE_NAME
+    lakehouse_timeseries_path = os.environ["LAKEHOUSE_ABFSS_PREFIX"] + "/Tables/" + os.environ["DPS_TABLE_NAME"]
     yield lakehouse_timeseries_path
     delta_table = get_ts_delta_table(azure_credential, lakehouse_timeseries_path)
     delta_table.delete()
 
 @pytest.fixture()
 def time_series(request, cognite_client):
+    sub_name = "testSubscription"
     timeseries_set = generate_timeseries_set(request.param)
+    remove_time_series_data(timeseries_set, cognite_client)
+    push_time_series_to_cdf(timeseries_set, cognite_client)
+    create_subscription_in_cdf(timeseries_set, sub_name, cognite_client)
```

Comment:
There may be a way for us to feed this subscription id into the config when running the replicator to resolve conflicts if two integration tests are running at the same time against the same cognite data-model

Reply:
After taking more of a look there may not be a clear cut way for now and this seems good for now, this is something we can work on later if needed.